### PR TITLE
bug 1426949 - reject crashes from certain browser configurations

### PR DESCRIPTION
**Pull request by willkg**

### CODE REVIEW

**Comment by willkg** [comment]
Things to verify:

1. Does the logic for `match_infobar_true` look right?
2. Are the two places in the code that we'll need to change when we do releases easy to identify and is it clear what that person will need to change?

Otherwise, this change is pretty straight-forward. Let me know if it's not.

File: tests/unittest/test_throttler.py [hunk 1]
```diff
@@ -90,6 +189,16 @@ def test_hangid(self):
         throttler = Throttler(ConfigManager.from_dict({}))
         assert throttler.throttle(raw_crash) == (REJECT, 'has_hangid_and_browser', None)
 
+    def test_infobar(self):
+        raw_crash = {
+            'ProductName': 'Firefox',
+            'SubmittedFromInfobar': 'true',
+            'Version': '52.0.2',
```
Comment (lonnen) [inline tests/unittest/test_throttler.py]: glad to see a more complex version number than major + minor

**Comment by lonnen** [comment]
r+

File: antenna/throttler.py [hunk 2]
```diff
@@ -178,6 +178,31 @@ def always_match(crash):
...
+        return False
+
+    return (
+        product == 'Firefox' and
+        infobar == 'true' and
+        version.startswith(('52.', '53.', '54.', '55.', '56.', '57.', '58.', '59.')) and
+        # FIXME(willkg): buildid is a string that starts with a YYYYMMDD date
+        # so we're doing string compares. Change this to the buildid date when
+        # we push out fixes.
+        buildid < '20180110'
```
Comment (lonnen) [inline antenna/throttler.py]: consider putting this a bit further out. 59 will go live 2018-03-13

Comment (lonnen) [reply antenna/throttler.py]: just in case

File: tests/unittest/test_throttler.py (hunk 1 above)
Comment (willkg) [reply tests/unittest/test_throttler.py]: I can sprinkle in a few more.

File: antenna/throttler.py (hunk 2 above)
Comment (willkg) [reply antenna/throttler.py]: I'll push it further out.

**Comment by willkg** [comment]
I forgot how much I like working on Antenna. CI runs so much faster.

**Comment by willkg** [comment]
I'm going to land this and verify it on -stage.

**Pull request merged**